examples/vpc_service_control/vpc_sc_project/main.tf

```diff
@@ -0,0 +1,318 @@
+  enable_flow_logs         = false
+  ip_cidr_range            = "10.7.0.0/16"
+  name                     = "vpc-subnet"
+  network                  = "${google_compute_network.test-vpc.self_link}"
+  private_ip_google_access = true
+  project                  = "${google_project.vpc_sc_network_project.project_id}"
+  region                   = "${var.cloud_router_region}"
+}
+
+
+
+
+/* STEP 2A: DELETE THIS COMMENT LINE TO RESERVE STATIC IP (costs ~$0.24/day) 
+resource "google_compute_address" "vpc_sc_vpn_ip" {
+  address_type = "EXTERNAL"
+  name         = "vpc-sc-vpn-ip"
+  network_tier = "PREMIUM"
+  project      = "${google_project.vpc_sc_network_project.project_id}"
+  region       = "${var.cloud_router_region}"
+}
+
+output "ip_addr_of_cloud_vpn_router" {
+  value = "${google_compute_address.vpc_sc_vpn_ip.address}"
+}
+ STEP 2B: DELETE THIS END-COMMENT LINE.  Then run terraform apply. */
+
+
+/* STEP 4A: DELETE THIS COMMENT LINE TO CREATE CLOUD VPN ROUTER (costs ~$1.80/day) 
+resource "google_compute_router" "vpc_sc_cloud_router" {
+  bgp {
+    advertise_mode = "DEFAULT"
+    asn            = "64513"
+  }
+
+  name    = "vpc-sc-cloud-router"
+  network = "${google_compute_network.test-vpc.self_link}"
+  project = "${google_project.vpc_sc_network_project.project_id}"
+  region  = "${var.cloud_router_region}"
+}
+
+resource "google_compute_vpn_gateway" "target_gateway" {
+  name    = "target-vpn-gateway"
+  network = "${google_compute_network.test-vpc.self_link}"
+  project = "${google_project.vpc_sc_network_project.project_id}"
+  region  = "${var.cloud_router_region}"
+}
+
+resource "google_compute_forwarding_rule" "fr_for_vpn_gateway" {
+  name        = "frforvpngateway"
+  ip_protocol = "ESP"
+  ip_address  = "${google_compute_address.vpc_sc_vpn_ip.address}"
+  target      = "${google_compute_vpn_gateway.target_gateway.self_link}"
+  project = "${google_project.vpc_sc_network_project.project_id}"
+  region = "${var.cloud_router_region}"
+}
+
+resource "google_compute_forwarding_rule" "fr_udp500" {
+  name        = "frforvpngatewayudp500"
+  ip_protocol = "UDP"
+  port_range = "500"
+  ip_address  = "${google_compute_address.vpc_sc_vpn_ip.address}"
+  target      = "${google_compute_vpn_gateway.target_gateway.self_link}"
+  project = "${google_project.vpc_sc_network_project.project_id}"
+  region = "${var.cloud_router_region}"
+}
+
+resource "google_compute_forwarding_rule" "fr_udp4500" {
+  name        = "frforvpngatewayudp4500"
+  ip_protocol = "UDP"
+  port_range = "4500"
+  ip_address  = "${google_compute_address.vpc_sc_vpn_ip.address}"
+  target      = "${google_compute_vpn_gateway.target_gateway.self_link}"
+  project = "${google_project.vpc_sc_network_project.project_id}"
+  region = "${var.cloud_router_region}"
+}
+
+resource "google_compute_vpn_tunnel" "vpc_sc_vpn_tunnel" {
+  ike_version             = "2"
+  name                    = "vpc-sc-vpn-tunnel"
+  peer_ip                 = "${var.ip_addr_of_onprem_vpn_router}"
```

Comment:
```suggestion
  peer_ip                 = "${google_compute_address.onprem_vpn_ip.address}"
```

Reply:
This doesn't work, per my comment above.

Reply:
Indeed. I should've added a comment saying this suggestion is assuming both VPNs are created in the same root config and won't work with your current setup.